src/resource_clients/key_value_store.ts

```diff
@@ -112,20 +112,21 @@ export class KeyValueStoreClient extends ResourceClient {
      * If the client has permission to access the key-value store's URL signing key,
      * the URL will include a signature which will allow the link to work even without authentication.
      *
-     * You can optionally control how long the signed URL should be valid using the `expiresInMillis` option.
-     * This value sets the expiration duration in milliseconds from the time the URL is generated.
+     * You can optionally control how long the signed URL should be valid using the `expiresInSecs` option.
+     * This value sets the expiration duration in seconds from the time the URL is generated.
      * If not provided, the URL will not expire.
      *
      * Any other options (like `limit` or `prefix`) will be included as query parameters in the URL.
      */
-    async createKeysPublicUrl(options: KeyValueClientListKeysOptions = {}, expiresInMillis?: number) {
+    async createKeysPublicUrl(options: KeyValueClientCreateKeysUrlOptions = {}) {
```

Comment:
I would rather keep the second argument as it is, and add a `log.deprecated` to it when the value is present, just in case it's already used somewhere. I can imagine it will be mostly testing actors from your end, but still, even for those, it feels like a better thing to have a proper warning than just a compilation failure.

We will be releasing major bumps for all the packages relatively soon (let's say by the end of this year), so we can remove it there.